backend/app/router/endpoints/notes.py

```
@@ -2,251 +2,85 @@
 
+class CreateNoteRequest(BaseModel):
+    title: str
+    tags: List[str]
+    cues: str
+    notes: str
+    summary: str
+    docs: List[str]
+    questions: List[Question | MultiSelectQuestion] = []
+
+@router.post("/", response_model=Note)
+def create_note(request: CreateNoteRequest):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)